scripts/generate/templates/EnumerableMap.js

```diff
@@ -57,14 +57,10 @@ import {EnumerableSet} from "./EnumerableSet.sol";
 /* eslint-enable max-len */
 
 const defaultMap = () => `\
-// To implement this library for multiple types with as little code
-// repetition as possible, we write it in terms of a generic Map type with
-// bytes32 keys and values.
-// The Map implementation uses private functions, and user-facing
-// implementations (such as Uint256ToAddressMap) are just wrappers around
-// the underlying Map.
-// This means that we can only create new EnumerableMaps for types that fit
-// in bytes32.
+// To implement this library for multiple types with as little code repetition as possible, we write it in
+// terms of a generic Map type with bytes32 keys and values. The Map implementation uses private functions,
+// and user-facing implementations (such as UintToAddressMap) are just wrappers around the underlying Map.
```

Comment:
“are just wrappers” -> “that are just wrappers”?

Reply:
> [...], and user-facing implementations (...) are just wrappers around the underlying Map.

This (current version) feels ok to me. @frangio ? Maybe we can remove the "(such as UintToAddressMap)" part.

Reply:
Yeah the current version is okay but I removed parentheses to make it less confusing.